pkg/services/ngalert/image/service.go

```diff
@@ -162,17 +151,3 @@ func (s *ScreenshotImageService) NewImage(ctx context.Context, r *models.AlertRu
 	image := result.(models.Image)
 	return &image, nil
 }
-
-// NotAvailableImageService is a service that returns ErrScreenshotsUnavailable.
-type NotAvailableImageService struct{}
```

Comment:
This is used when images are disabled, I don't think it can go in `testing.go`.

Reply:
Good point, I'll look for a better way to represent this.

Reply:
Hmm, I only see two references to this type, both in the tests for the state manager. :thinking: Maybe there's a third implementation for when images are disabled?

Reply:
Ah - when we disable images, we still instantiate the same `image.ScreenshotImageService`, but it wraps the `screenshot.ScreenshotUnavailableService` underneath. `NotAvailableImageService` is only used for tests:

https://github.com/grafana/grafana/blob/ed641339430333e2fbfc0ca5bb5c63504173ee17/pkg/services/ngalert/image/service.go#L87